pygaggle/reader/dpr_reader.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+from transformers import DPRReader, DPRReaderTokenizer
+
+from .base import Reader, Answer
+from ..reranker.base import Query, Text
+
+class DPRReader(Reader):
+    def __init__(
+        self,
+        model_name: str,
+        tokenizer_name: str = None,
+        num_spans: int = 16,
```

Comment:
can you add a bit of a description for all the function parameters, some of these might not be 100% apparent to non-QA folks